lib/money/bank/variable_exchange.rb

```diff
@@ -265,6 +265,10 @@ def import_rates(format, s, opts = {})
           warn '[WARNING] Using :ruby format when importing rates is potentially unsafe and ' \
             'might lead to remote code execution via Marshal.load deserializer. Consider using ' \
             'safe alternatives such as :json and :yaml.'
+        elsif format == :yaml
+          warn '[WARNING] Using :yaml format when importing rates is potentially unsafe and ' \
+            'might lead to remote code execution via Marshal.load deserializer. Consider using ' \
+            'safe alternatives such as :json and :ruby.'
```

Comment:
Hi, I guess that `:ruby` is not safe either, so this should advise to use `json` only